components/FleetClusters.vue

```diff
@@ -62,7 +62,7 @@ export default {
     },
 
     pagingParams() {
-      const inStore = this.$store.getters['currentProduct'].inStore;
+      const inStore = this.$store.getters['currentStore'](FLEET.CLUSTER);
```

Comment:
Prior to this change the assumption was that all resources within a product used the same store. 

Because projects are within the management store but within the explorer product that was no longer true. This allows you to look up the store for a specific resource or default to the current product store.

Reply:
Ok, think i'm getting there. I thought management resources were 'global' and only in the `local` cluster... whilst projects are cluster specific and therefor ok in the cluster/explorer context. Looks like though they only exist in the local cluster but there's some magic to show them in non-local clusters?

Reply:
Honestly I hadn't given it that much thought. This is the most I've worked with management so far so it might be worth talking to Vince about it today. I don't think it's local and this is actually based on the reminder Vince gave on Tuesday. We need to check if the cluster `isRancher` before showing projects since it's a rancher construct.

Here the only magic that's going on is breaking the original assumption that all resources within a product will use the same module/store.